### PR TITLE
Project Loved: Week of February 25th

### PR DESCRIPTION
**Pull request by cl8n**

### CODE REVIEW

File: news/2018-02-25-project-loved-week-of-february-25th.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+layout: post
+title: "Project Loved: Week of February 25th"
+date: 2018-02-25 18:00:00 +0000
+---
+
+The [Project Loved](https://osu.ppy.sh/community/forums/120) voting schedule is changing slightly starting this week! Come check out the new picks for all four modes now!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Hello everyone! Welcome to the last [Project Loved](https://osu.ppy.sh/community/forums/120) showcase of February 2018! We are slightly changing up the voting format starting this week based on feedback from the community. This change will simplify the Loved voting schedule, as well as make it a lot easier to follow for those who care about more than just one gamemode!
```
Comment (MegaApplePi) [inline news/2018-02-25-project-loved-week-of-february-25th.md]: `[...] as well as make it [...]`
this reads weird to me, but I'm not sure on how to rephrase it :/

File: news/2018-02-25-project-loved-week-of-february-25th.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+Each mode will have a vote every 2 weeks, all at the same time as opposed to the staggered structure we previously used. Each vote will have the following amount of maps:
+
+- 10 osu!
```
Comment (MegaApplePi) [inline news/2018-02-25-project-loved-week-of-february-25th.md]: maybe a hyphen in between the count and the game mode? or game mode, colon, space, then count?

File: news/2018-02-25-project-loved-week-of-february-25th.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+### Navigation
+
+[osu!](#osu)  
+[osu!taiko](#osutaiko)  
+[osu!catch](#osucatch)  
+[osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/vRyDI07FcTg" frameborder="0" allowfullscreen></iframe>
```
Comment (MegaApplePi) [inline news/2018-02-25-project-loved-week-of-february-25th.md]: last time when @mangomizer tried this, it broke... 
`<iframe width="560" height="315" src="https://www.youtube.com/embed/vRyDI07FcTg" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>`

(only going to mention this once) ¯\\_(ツ)\_/¯

File: news/2018-02-25-project-loved-week-of-february-25th.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+
+[![](/wiki/shared/news/2018-02-25-project-loved-week-of-february-25th/mania/river-flows-in-you.jpg)](https://osu.ppy.sh/community/forums/topics/708941)
+
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+This set is something special. Brought by our own Elite Mapper [LordRaika](https://osu.ppy.sh/users/3265023), it is a mapset that really nicely conveys what kind of a charter LordRaika is.
+
+Not only is this mapset keysounded, the song itself is LordRaika's mix of it, and it also has a wonderful storyboard by the elite storyboarder [Exile-](https://osu.ppy.sh/users/2559349).
+
+It doesn't end there! Because the mapset also offers an immense set of 30 difficulties throughout all 6 rankable keymodes, from 4k to 9k, offering a wide range of patterns and densities to train your favourite keymode or perhaps, learn a new one!
```
Comment (MegaApplePi) [inline news/2018-02-25-project-loved-week-of-february-25th.md]: I assume that this was a copy-paste thing from the people who actually wrote it, but the casing of the K's for the key counts are inconsistent

File: news/2018-02-25-project-loved-week-of-february-25th.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+
+Professional nerd [Nivrad00](https://osu.ppy.sh/users/1984634) with another masterpiece which twists the boss rush theme made prominent in the rhythm game scene via Renard's MungYoDance series, as to accompany his own compositions. Thirteen different bosses which each test your abilities to an ultimate, with attack after pummelling attack of dense patterning, rapid-fire streams and flexible fingerwork.
+
+It is also Nivrad00's 100th completed chart in his rhythm game charting career, which is certainly a milestone to celebrate.
+
+So good luck with it, and as always, have fun!
+
+---
+
+From everyone on the Project Loved team, we hope you enjoy this week's selection, and look forward to the next! The winning maps will hit the [beatmap listing](https://osu.ppy.sh/p/beatmaplist&s=4&r=12) sometime during the weekend and will be announced via the [osugame twitter](https://twitter.com/osugame).
```
Comment (MegaApplePi) [inline news/2018-02-25-project-loved-week-of-february-25th.md]: beatmap listing link -> `https://osu.ppy.sh/beatmapsets?m=-1&s=8`
?

File: news/2018-02-25-project-loved-week-of-february-25th.md [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+Each mode will have a vote every 2 weeks, all at the same time as opposed to the staggered structure we previously used. Each vote will have the following amount of maps:
+
+- osu! - 10
+- osu!taiko - 7
+- osu!catch - 5
+- osu!mania - 10
+
+This new system will offer a lot more breathing room to both the community and the Loved captains, as well as make the voting itself more uniform! If you have any comments, questions, or concerns regarding the new system, be sure to leave them in the comments section of this post.
+
+With that out of the way, let's get on to this week's picks! For this round, [Strato](https://osu.ppy.sh/users/6492199) has made us some awesome showcase videos for each mode. Check them out at the beginning of each section if you prefer a video format!
```
Comment (TPGPL) [inline news/2018-02-25-project-loved-week-of-february-25th.md]: for us

File: news/2018-02-25-project-loved-week-of-february-25th.md [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+date: 2018-02-25 18:00:00 +0000
+---
+
+The [Project Loved](https://osu.ppy.sh/community/forums/120) voting schedule is changing slightly starting this week! Come check out the new picks for all four modes now!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Hello everyone! Welcome to the last [Project Loved](https://osu.ppy.sh/community/forums/120) showcase of February 2018! We are slightly changing up the voting format starting this week based on feedback from the community. This change will simplify the voting schedule and make it a lot easier to follow for those who care about more than just one gamemode!
+
+Each mode will have a vote every 2 weeks, all at the same time as opposed to the staggered structure we previously used. Each vote will have the following amount of maps:
```
Comment (TPGPL) [inline news/2018-02-25-project-loved-week-of-february-25th.md]: Of beatmaps

File: news/2018-02-25-project-loved-week-of-february-25th.md [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/tstFskmG0MM?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+---
+
+[![](/wiki/shared/news/2018-02-25-project-loved-week-of-february-25th/taiko/los-los-los.jpg)](https://osu.ppy.sh/community/forums/topics/708912)
+
+*written by [Nwolf](https://osu.ppy.sh/users/1910766)*
+
+A bit of a normal break from all the insanity, there isn't much to actually say about [Los! Los! Los!](https://osu.ppy.sh/beatmapsets/603361) other than it being an outstanding anime map in the sea of them. Long, energetic bursts of energy characterize this map as a force, with very dense sections of 1/4 triplets. Los! captures our hearts in full.
```
Comment (TPGPL) [inline news/2018-02-25-project-loved-week-of-february-25th.md]: A bit from?

Comment (cl8n) [reply news/2018-02-25-project-loved-week-of-february-25th.md]: I think it's correct as-is

it's "a break from ..." with some extra stuff

**Pull request merged**